src/loaders/TextureLoader.js

```diff
@@ -7,10 +7,12 @@ import { ImageLoader } from './ImageLoader.js';
 import { Texture } from '../textures/Texture.js';
 import { Loader } from './Loader.js';
 
-function TextureLoader( manager ) {
+function TextureLoader( manager, imageLoader ) {
```

Comment:
Instead of enhancing the `ctor`, I suggest to introduce `setImageLoader()` since the library uses this pattern in other loaders, too (e.g. `setDRACOLoader()` or `setDDSLoader()`).

Reply:
Done! Thanks for the suggestion, it makes the code a little bit cleaner all around in my opinion.